modules/sdk/gradle-plugins-workspace/src/main/java/com/liferay/gradle/plugins/workspace/docker/DockerListImage.java

```diff
@@ -32,9 +32,9 @@
 /**
  * @author Seiphon Wang
  */
-public class DockerFindDanglingImage extends AbstractDockerRemoteApiTask {
+public class DockerListImage extends AbstractDockerRemoteApiTask {
```

Comment:
I think this should be `DockerListDanglingImage`

Reply:
oh i see, people would need to add `dockerListImage { dangling = true }`?  Is that it?

Reply:
Yes, it is